e2e/cases/doctor-rspack/plugin.test.ts

```diff
@@ -155,9 +169,11 @@ test('rspack data store', async () => {
     parsedSize: 0,
   });
   expect(graphData.modules[0].path).toMatch('/fixtures/a.js');
+
+  // TODO: Change report Rspack config to afterPlugin hook, this should be reWrite
   // @ts-ignore
-  const ruleLengthList = configs[0].config.module?.rules?.map(
-    (_rule) => (_rule as RuleSetRule)?.use?.length,
-  );
-  expect(ruleLengthList).toEqual([1, 3, 3, 3]);
+  // const ruleLengthList = configs[0].config.module?.rules?.map(
```

Comment:
this test will be modified by @easy1090 later

Reply:
ok~